packages/hub/config/structure.sql

```diff
@@ -2,8 +2,8 @@
 -- PostgreSQL database dump
```

Comment:
@IanCal this file is used to initialise db only. I don't think there is any consequence

Reply:
yarn db:test-init